### ISSUE TITLE
Post request configuration help😟😟

### ISSUE THREAD

**Issue by daya0576** [sanathp/statusok#21]
Here is my **post request body**:
`{"data": [{"password": "xxx", "email": "xxx"}]}`
![image](https://user-images.githubusercontent.com/6239652/32533256-44bbfaba-c415-11e7-936d-70ffe5c06576.png)

I tried to config in this way:
![image](https://user-images.githubusercontent.com/6239652/32533340-c2359032-c415-11e7-9d2a-ea25f82ca4fa.png)

But failed and got this error:
![image](https://user-images.githubusercontent.com/6239652/32533277-643a89f6-c415-11e7-97ce-fdd2f9772c3c.png)

**Comment by sanathp** [comment]
Sorry, currently support of complex JSON as body dosn't exist. Only key value pairs are accepted for now.
Will add support to all types of JSON in next release.

**Comment by faisalbasra** [comment]
Any update on this issue?

**Comment by sanathp** [comment]
Yes the above format is possible. Currently only key-value pairs are supported.

**Comment by SciLor** [comment]
You would only need to support raw format.
